misc_docs/syntax/extension_raw_expression.mdx

```diff
@@ -0,0 +1,34 @@
+---
+id: "raw-expression"
+keywords: ["javascript", "raw", "expression"]
+name: "%raw"
+summary: "This is the `JavaScript expression` extension point."
+category: "extensionpoints"
+---
+
+`%raw` is used to embed JavaScript expressions.
```

Comment:
Maybe this makes it clearer why it's called `raw`.. we want to embed `raw JS` basically.

```suggestion
`%raw` is used to embed "raw JavaScript expressions".
```